pkgs/context-labs/mactop/registry.yaml

```diff
@@ -0,0 +1,19 @@
+packages:
+  - type: github_release
+    repo_owner: context-labs
+    repo_name: mactop
+    description: mactop - Apple Silicon Monitor Top written in pure Golang! Under 1,000 lines of code
+    asset: mactop_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
+    checksum:
+      type: github_release
+      asset: checksums.txt
+      algorithm: sha256
+    supported_envs:
+      - darwin/arm64
+    version_constraint: semver("> 0.1.2")
+    version_overrides:
+      - version_constraint: semver("<= 0.1.2")
```

Comment:
Hmm, it seems you don't use `cmdx s`.

Reply:
I once ran `cmdx s` then fix it by hand

Reply:
Ah, did you update the commit by `git commit --amend` or `git rebase` or somehow?
`cmdx s` generates a commit but the commit 7e550e236ae145760419171a093b36a9054b083d looks different from it.
If so, please don't do that from the next time.
`cmdx s` creates a commit to distinguish the code `cmdx s` generates from the code users write by hand.
ref. https://aquaproj.github.io/docs/products/aqua-registry/changelog#2024-05-24

Reply:
I found the style guide is stale.
I'll update it.

https://aquaproj.github.io/docs/develop-registry/registry-style-guide/#version_overrides-style-guide
https://github.com/aquaproj/aquaproj.github.io/blob/32d2656dafcc9af44c6bae73eeae6e94adf3c6de/docs/develop-registry/registry-style-guide.md?plain=1#L102-L111

Reply:
- https://github.com/aquaproj/aquaproj.github.io/pull/996